scaleway/provider_test.go

```diff
@@ -67,6 +67,7 @@ func getHTTPRecoder(t *testing.T, update bool) (client *http.Client, cleanup fun
 		i.Request.Headers = i.Request.Headers.Clone()
 		delete(i.Request.Headers, "x-auth-token")
 		delete(i.Request.Headers, "X-Auth-Token")
+		delete(i.Request.Headers, "Authorization")
```

Comment:
Why is this change required? I thought is was needed by #1078

Reply:
Yes, I've just removed from here